web/src/main/java/org/mskcc/cbio/oncokb/api/pvt/PrivateSearchApiController.java

```diff
@@ -757,3 +808,44 @@ public int compare(LevelOfEvidence o1, LevelOfEvidence o2) {
     }
 }
 
+class GeneCount {
```

Comment:
Do we want this in a separate file?

Reply:
Not necessarily if they are not used anywhere else.